CODEOWNERS

```diff
@@ -14,6 +14,7 @@ azure-pipelines.yml  @openvinotoolkit/openvino-admins
 
 # IE Core:
 /inference-engine/  @openvinotoolkit/openvino-ie-maintainers
+/inference-engine/ie_bridges/python @akuporos @nkogteva
```

Comment:
probably @openvinotoolkit/openvino-tools-maintainers can be used here

Reply:
What about created team @openvinotoolkit/openvino-ie-python-api-maintainers?
It'll be deleted?

Reply:
I used @openvinotoolkit/openvino-ie-python-api-maintainers